README.md

```diff
@@ -306,6 +306,87 @@ spec:
+
+For instance the `konveyor.openshift.io_v1alpha1_velero_cr.yaml` file might look something like this:
+
+```
+apiVersion: konveyor.openshift.io/v1alpha1
+kind: Velero
+metadata:
+  name: example-velero
+spec:
+  use_upstream_images: true
+  olm_managed: true
+  default_velero_plugins:
+  - aws
+```
+
+When the installation is succeeded, create a Velero CR
+```
+oc create -f deploy/crds/konveyor.openshift.io_v1alpha1_velero_cr.yaml
+```
+
+Post completion of all the above steps, you can check if the operator was successfully installed or not, the expected result for the command `oc get all -n oadp-operator` is as follows:
+```
+NAME                                             READY   STATUS    RESTARTS   AGE
+pod/oadp-default-aws-registry-568978c9dc-glpfj   1/1     Running   0          10h
+pod/oadp-operator-64f79d9bf4-4lzl9               1/1     Running   0          10h
+pod/restic-bc5tm                                 1/1     Running   0          10h
+pod/restic-dzrkh                                 1/1     Running   0          10h
+pod/restic-z4mhx                                 1/1     Running   0          10h
+pod/velero-779f785b7d-5z6qf                      1/1     Running   0          10h
+
+NAME                                    TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)             AGE
+service/oadp-default-aws-registry-svc   ClusterIP   172.30.155.164   <none>        5000/TCP            10h
+service/oadp-operator-metrics           ClusterIP   172.30.58.121    <none>        8383/TCP,8686/TCP   10h
+
+NAME                    DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR   AGE
+daemonset.apps/restic   3         3         3       3            3           <none>          10h
+
+NAME                                        READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/oadp-default-aws-registry   1/1     1            1           10h
+deployment.apps/oadp-operator               1/1     1            1           10h
+deployment.apps/velero                      1/1     1            1           10h
+
+NAME                                                   DESIRED   CURRENT   READY   AGE
+replicaset.apps/oadp-default-aws-registry-568978c9dc   1         1         1       10h
+replicaset.apps/oadp-operator-64f79d9bf4               1         1         1       10h
+replicaset.apps/velero-779f785b7d                      1         1         1       10h
+
+NAME                                                       HOST/PORT                                                                                        PATH   SERVICES                        PORT       TERMINATION   WILDCARD
+route.route.openshift.io/oadp-default-aws-registry-route   oadp-default-aws-registry-route-oadp-operator.apps.cluster-dshah-4-5.dshah-4-5.mg.dog8code.com          oadp-default-aws-registry-svc   5000-tcp                 None
+``` 
+
 ### Cleanup
```

Comment:
We should mention that for OLM uninstalling and unsubscribing from the operator is sufficient.

Reply:
Same, we don't use web console for Velero CR, so for cleaning up we need to delete CR by following cleanup steps.

Reply:
My point is that a user *should* be using the web console for creating the Velero CR. This is the point of specifying the example CR in OLM so that a user can click on the API and create a default instance that will just work out of the box.